### PR TITLE
Visual Studio 2015 dark style

### PR DESCRIPTION
**Pull request by Nicolas01**
A new style inspired from Visual Studio 2015.

### CODE REVIEW

File: src/styles/vs2015.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  color: #569CD6;
+}
+.hljs-link {
+  color: #569CD6;
+  text-decoration: underline;
+}
+
+.hljs-built_in,
+.hljs-type,
+.css .hljs-number {
```
Comment (isagalaev) [inline src/styles/vs2015.css]: There shouldn't be any language-specific rules in a style. One of our features is consistency across languages, and we are happy to sacrifice similarities to what the original style (Visual Studio in this case) does. Could you remove all the language specifiers from the file, please.

File: src/styles/vs2015.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  background-color: #144212;
+  display: inline-block;
+  width: 100%;
+}
+
+.hljs-deletion {
+  background-color: #600;
+  display: inline-block;
+  width: 100%;
+}
```
Comment (isagalaev) [inline src/styles/vs2015.css]: A couple of things that are missing: `.hljs-template-tag` and `.hljs-template-variable`. Stick those somewhere in as well :-)

**Comment by Nicolas01** [comment]
I added the requested changes.

**Comment by isagalaev** [comment]
One more thing: what's your preferred name to mention it in the contributor list (AUTHORS.en.txt) and CHANGES.md? :-)

**Comment by Nicolas01** [comment]
Here is the name to mention: Nicolas LLOBERA  
Should I add it myself to AUTHORS.en.txt and CHANGES.md?

**Comment by isagalaev** [comment]
Please be so kind! Will save me some fuss when merging.

**Comment by isagalaev** [comment]
Merged. Thanks for your contribution!

**Pull request merged**